scss/components/card.scss

```diff
@@ -128,7 +128,7 @@ $block: ns(card);
     &__header {
         color: $fd-color;
         @include fd-type(1);
-        margin-bottom: 1px;
+        margin-bottom: fd-space(xs);
```

Comment:
This is too big if you compare to the design comps. Use 4px which is closer to the design.

`margin-bottom: fd-space(xxs);`